x-pack/plugins/reporting/jest.integration.config.js

```diff
@@ -9,4 +9,7 @@ module.exports = {
   preset: '@kbn/test/jest_integration',
   rootDir: '../../..',
   roots: ['<rootDir>/x-pack/plugins/reporting'],
+  collectCoverageFrom: [
+    '<rootDir>/x-pack/plugins/reporting/server/export_types/printable_pdf_v2/**/*.{js,ts,tsx}',
+  ],
```

Comment:
I think this should be reset from `main`

`git checkout main x-pack/plugins/reporting/jest.integration.config.js`